README.md

```diff
@@ -10,13 +10,82 @@ Requirements
 ------------
 
 * Django 1.6
-* django-localflavor
-* django-databrowse
 
 
 Installation
 ------------
 
+1. Install from GitHub.
+    ```sh
+        $ pip install git+git://github.com/unt-libraries/django-controlled-vocabularies.git
+    ```
+
+2. Add app and dependency to INSTALLED_APPS.
+    ```python
+        INSTALLED_APPS = (
+            'django_databrowse',
+            'accession',
+        )
+    ```
+
+3. Include the URLs.
+    ```python
+        urlpatterns = [
+            url(r'^admin/', include(admin.site.urls)),
+            url(r'^tfc/', include('accession.urls')),
+        ]
+    ```
+
+4. Sync the database and create a superuser account.
+    ```sh
+        $ python manage.py syncdb
+    ```
+
+
+Developing
+----------
+
+1. [Install Docker](http://docs.docker.com/installation/)
+
+2. Install Docker Compose
+    ```sh
+        $ pip install docker-compose
+    ```
+
+3. Clone the repository.
+    ```sh
+        $ git clone https://github.com/unt-libraries/django-controlled-vocabularies
+    ```
+
+4. Build the images.
+    ```sh
+        $ docker-compose build
+    ```
+
+5. Allow the database to initialize everything (only needed the first time the image is loaded).
+    ```sh
+        $ docker-compose up db
+    ```
+    Wait until the output stops, then execute the following to stop the container.
+    ```sh
+        $ docker-compose stop db
+    ```
```

Comment:
What do you think if we changed step 5 to

```markdown
5. Warm-up the database. It will take 10-15 seconds to complete after it has been pulled from the Docker Hub.
    ```sh
        $ docker-compose up -d db
    ```